pkg/cli/admin/admin.go

```diff
@@ -90,6 +90,8 @@ func NewCommandAdmin(f kcmdutil.Factory, streams genericiooptions.IOStreams) *co
 				copytonode.NewCmdCopyToNode(f, streams),
 				rebootmachineconfigpool.NewCmdRebootMachineConfigPool(f, streams),
 				waitfornodereboot.NewCmdWaitForNodeReboot(f, streams),
+				// Will uncomment when the new command implementation will be ready
+				// nodeimage.NewCmdNodeImage(f, streams),
```

Comment:
No need to modify this file for this. We can add `nodeimage.NewCmdNodeImage(f, streams),` in next PR?